libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.service.ts

```diff
@@ -1244,28 +1320,46 @@ export class ParentalLeaveService extends BaseTemplateApiService {
     return undefined
   }
 
-  async sendApplication({ application }: TemplateApiModuleActionProps) {
+  async sendApplication({ application, params }: TemplateApiModuleActionProps) {
```

Comment:
Do we need to put default value on params variable? There is few call which doesn't include params ( I have not test the application and not sure if it needed )

Reply:
Yes, I think we should put default on it. 👍

Reply:
I think we should